src/python_testing/TC_SWTCH.py

```diff
@@ -424,12 +455,12 @@ def steps_TC_SWTCH_2_4(self):
                 TestStep(2, "Set up subscription to all events of Switch cluster on the endpoint"),
```

Comment:
```suggestion
                TestStep(2, "Set up subscription to all events and attributes of Switch cluster on the endpoint"),
```
Also need to update this in test-plan spec.

Reply:
Test plan update: https://github.com/CHIP-Specifications/chip-test-plans/pull/4599